### ISSUE TITLE
Manueller Modus automatisch zurücksetzten

### ISSUE THREAD

**Issue by FI84** [rg-engineering/ioBroker.heatingcontrol#196]
Beim manuellen wieder zurückstellen auf den Profil Wert könnte man auch wenn Profil Wert = Soll Wert, dann Reset manuel machen

**Comment by rg-engineering** [comment]
nein, ein automatisches Zurücksetzen des manuellen Modus kann ich nicht implementieren. Es gibt zu viele Situationen und zu unterschiedliches Verhalten der Thermostate, in denen es nicht funktioniert...

**Comment by FI84** [comment]
Vielleicht könnte man es im Adapter auswählen ob eine "automatische Zurücksetzung bei Profil Wert = Soll Wert" ausgeführt werden soll
